CMakeLists.txt

```
@@ -1,55 +1,68 @@
-################################################################################
+# ##############################################################################
 # Preamble
 
 cmake_minimum_required(VERSION 3.17)
 
 project(
   tuv-x
-  VERSION 0.9.0
-  LANGUAGES Fortran CXX C
-)
+  VERSION 0.8.0
+  LANGUAGES Fortran CXX C)
 
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
 set(CMAKE_MODULE_PATH "${CMAKE_MODULE_PATH};${PROJECT_SOURCE_DIR}/cmake")
 
-set(CMAKE_USER_MAKE_RULES_OVERRIDE "${PROJECT_SOURCE_DIR}/cmake/set_defaults.cmake")
+set(CMAKE_USER_MAKE_RULES_OVERRIDE
+    "${PROJECT_SOURCE_DIR}/cmake/set_defaults.cmake")
 
 if(NOT CMAKE_BUILD_TYPE)
-  set(CMAKE_BUILD_TYPE "Release" CACHE STRING
-      "Choose the type of build, options are: Debug Release RelWithDebInfo MinSizeRel."
-      FORCE)
+  set(CMAKE_BUILD_TYPE
+      "Release"
+      CACHE
+        STRING
+        "Choose the type of build, options are: Debug Release RelWithDebInfo MinSizeRel."
+        FORCE)
```

Not really sure how to best handle the formatting but I think this would be a nice middle ground and uses the built-in list of build types (https://cmake.org/cmake/help/latest/variable/CMAKE_CONFIGURATION_TYPES.html).
```suggestion
  set(CMAKE_BUILD_TYPE "Release"
      CACHE STRING
        "Choose the type of build, options are: ${CMAKE_CONFIGURATION_TYPES}."
      FORCE)
```